python-sdk/dev/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN apt-get install -y --no-install-recommends \
         libsasl2-modules \
         freetds-dev \
         libssl-dev \
-        libkrb5-dev
+        libkrb5-dev \
+        libmariadb-dev
```

Comment:
@phanikumv thanks a lot for looking into this! 
What is the benefit of installing the maria DB package as opposed to `libmysqlclient-dev`?

Reply:
libmysqlclient-dev is not available, hence used the alternative.


```
#0 0.536 Package libmysqlclient-dev is not available, but is referred to by another package.
#0 0.536 This may mean that the package is missing, has been obsoleted, or
#0 0.536 is only available from another source
#0 0.536 However the following packages replace it:
#0 0.536   libmariadb-dev-compat libmariadb-dev
#0 0.536 
#0 0.583 E: Package 'libmysqlclient-dev' has no installation candidate

```